zappa/core.py

```diff
@@ -982,9 +989,13 @@ def get_manylinux_wheel_url(self, package_name, package_version):
 
         for f in data["releases"][package_version]:
             if re.match(self.manylinux_wheel_file_match, f["filename"]):
-                return f["url"], f["filename"]
+                # Since we have already lowered package names in get_installed_packages
+                # manylinux caching is not working for packages with capital case in names like MarkupSafe
+                return f["url"], f["filename"].lower()
```

Comment:
Looks like a testcase is missing for this, can you add one?

Reply:
added test